ow_lander/urdf/lander.xacro

```diff
@@ -920,7 +920,7 @@
     <axis
       xyz="0 0 -1" />
     <dynamics damping="50.0" />
-    <limit effort="100.0" velocity="0.2" />
+    <limit effort="100.0" velocity="0.2" lower="-3.14" upper="3.14"/>

```

Comment:
Can the tilt limits be tightened up to prevent collisions with the rover. We might not be able to prevent all collisions this way, but something like -pi/2 to pi/2 would probably be better.

Reply:
The tilt range we decided on a while back is in fact [-pi/2, pi/2] .

Reply:
Done!